docs/references/dimensions.mdx

```diff
@@ -76,11 +76,16 @@ models:
           dimension:
             type: date
           additional_dimensions:
-            dimension_name_one:
+            day_of_week_num:
               type: number
-              label: 'Day of Week'
+              label: 'Day of Week (number)'
               sql: 'day_of_week(${date})'
-            dimension_name_two:
+            day_of_week_day:
+              type: string
+              label: 'Day of Week (day)'
+              sql: ${date}
+              format: 'dddd'
```

Comment:
nice

Reply:
I don't think this works because the format expression is for a date value but the dimension is a string.